src/components/countries/CitySmallCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, useEffect, useState } from 'react';
+import { defaultCurrentWeather } from 'components/main/defaultStates';
+import { CurrentWeatherType } from 'types/weatherTypes';
+import { getWeather } from 'services/getWeather';
+import { getImgURL } from 'helpers';
+import Loader from '../../pictures/loader.gif';
+
+type SmallCardProps = {
+  lon: number;
+  lat: number;
+  cityName: string;
+};
+
+export const CitySmallCard: FC<SmallCardProps> = ({ lon, lat, cityName }) => {
+  const [currentWeather, setCurrentWeather] = useState<CurrentWeatherType>(defaultCurrentWeather);
+
+  useEffect(() => {
+    getWeather<{ current: CurrentWeatherType }>('/current/', lon, lat).then(res =>
+      setCurrentWeather(res.current)
+    );
+  }, []);
+  return (
+    <>
+      {currentWeather.time ? (
```

Comment:
why do you need to check `time` property here? can't you set `currentWeater` undefined as default and use condition `currentWeather ? (...)` ?